vertical-pod-autoscaler/pkg/utils/metrics/quality/quality_test.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package quality
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPrependBuckets(t *testing.T) {
+	assert.Equal(t, []float64{0.}, prependNegativeBuckets([]float64{}))
+	assert.Equal(t, []float64{-1., 0., 1}, prependNegativeBuckets([]float64{1.}))
+	assert.Equal(t, []float64{-2., -1., -0.5, -0.1, 0., 0.1, 0.5, 1., 2.}, prependNegativeBuckets([]float64{0.1, 0.5, 1., 2.}))
+	assert.Panics(t, func() { prependNegativeBuckets([]float64{-1.}) })
+}
+
+func TestGetExponentialBuckets(t *testing.T) {
+	assert.Equal(t, []float64{1., 2., 4.}, getExponentialBuckets(1., 5., 2.))
```

Comment:
The fact that 5.0 value seems ignored (unused) feels wrong.

Reply:
Removed funtion.